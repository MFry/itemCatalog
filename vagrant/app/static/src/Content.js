import React from 'react';


const Content = () => (


  <div className="container">
    <div className="section">

      {/* Icon Section   */}
      <div className="row">
        <div className="col s12 m4">
          <div className="icon-block">
            <h2 className="center light-blue-text" />
            <h5 className="center">Categories</h5>

            <p className="light">We did most of the heavy lifting for you to provide a default stylings that incorporate our custom components. Additionally, we refined animations and transitions to provide a smoother experience for developers.</p>
          </div>
        </div>

        <div className="col s12 m8">
          <div className="icon-block">
            <h2 className="center light-blue-text" />
            <h5 className="center">Latest Items</h5>

            <p className="light">By utilizing elements and principles of Material Design, we were able to create a framework that incorporates components and animations that provide more feedback to users. Additionally, a single underlying responsive system across all platforms allow for a more unified user experience.</p>
          </div>
        </div>
      </div>

    </div>
    <br /><br />
  </div>
);

export default Content;
