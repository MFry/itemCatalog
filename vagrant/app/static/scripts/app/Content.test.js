import React from 'react';
import Content from './Content';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';


describe('Testing basic <Content /> functionality', () => {
   it('should render without throwing an error', ()=> {
      expect(shallow(<Content />).prop('className')).toEqual("row");
   });
});

test('Content left panel renders', () => {
   const left_panel = renderer.create(
       <Content />
   );
    expect(left_panel).toMatchSnapshot();
});
