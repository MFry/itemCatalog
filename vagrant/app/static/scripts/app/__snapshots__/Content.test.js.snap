// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content left panel renders 1`] = `
<div
  className="row"
>
  <div
    className="col-md-4"
  >
    <h1>
      Categories
    </h1>
  </div>
  <div
    className="col-md-8"
  >
    <h1>
      Latest Items
    </h1>
  </div>
</div>
`;
